src/north/amazon/AmazonS3.class.js

```diff
@@ -51,6 +51,8 @@ class AmazonS3 extends ApiHandler {
     this.canHandleFiles = true
   }
 
+  static category = 'File'
```

Comment:
static variable are usually on the top of a class. Can you modify it for all the files please ?